roles/misc-perftools/templates/kvm_virsh.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
Please add comments indicating that this is not ours, and give credits to the author(s). If their project had a license, add the license as comment